### PR TITLE
[DPE-2078] Add scalability tests and resources cleanup scripts

### PR DESCRIPTION
**Pull request by dmitry-ratushnyy**
## Issue
In order to validate scalability of the MongoDB charm we needed integration tests
what will scale up\down cluster for different amount of units and also scale
down cluster including removal of MongoDB primary 

## Solution
Added test and script to cleanup resources

## Related issue
DPE-2078

### CODE REVIEW

**Review by delgod — CHANGES_REQUESTED** [review]

File: tests/integration/cleanup_resources.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    env["KUBECONFIG"] = os.path.expanduser("~/.kube/config")
+
+    output = subprocess.check_output(
+        f"tests/integration/ha_tests/scripts/destroy_chaos_mesh.sh {namespace}",
+        shell=True,
+        env=env,
+    )
+    logger.info("Destroy chaos mesh output:")
+    for line in output.decode("utf-8").splitlines():
+        logger.info(line)
```
[the remark this answers is not in the document]
Comment (dmitry-ratushnyy) [reply tests/integration/cleanup_resources.py:27]: oh, missed that one

File: tests/integration/ha_tests/test_ha.py [hunk 2]
```diff
@@ -651,3 +653,67 @@ async def test_network_cut(ops_test, continuous_writes):
...
+        logger.info(f"Scaling down by {abs(count)} units")
+        # find leader unit
+        leader_unit = await helpers.find_unit(ops_test, leader=True)
+        units_to_remove = []
+        for unit in ops_test.model.applications[app].units:
+            if not remove_leader and unit.name == leader_unit.name:
+                continue
+            else:
+                if len(units_to_remove) < abs(count):
+                    units_to_remove.append(unit.name)
```
Comment (Mehdi-Bendriss) [inline tests/integration/ha_tests/test_ha.py:698]:
```suggestion
            if len(units_to_remove) < abs(count):
                units_to_remove.append(unit.name)
```

File: tox.ini [hunk 3]
```diff
@@ -17,6 +17,7 @@ set_env =
     PYTHONPATH = {tox_root}/lib:{[vars]src_path}
     PYTHONBREAKPOINT=ipdb.set_trace
     PY_COLORS=1
+    PYTHONDONTWRITEBYTECODE=1
```
Comment (Mehdi-Bendriss) [inline tox.ini:20]: Why is this needed?

**Review by Mehdi-Bendriss — APPROVED** [review]
LGTM. Just a suggestion and a question

File: tox.ini (hunk 3 above)
Comment (dmitry-ratushnyy) [reply tox.ini:20]: To disable creating bytecode during tests.  Having bytecode created during testing leads to issues when debuging
